test/estimation.jl

```diff
@@ -19,3 +19,54 @@ using DSP, Test
     frequencies_estimated = sort(esprit(x, M, p, Fs))
     @test isapprox(frequencies', frequencies_estimated; atol = 1e-2)
 end
+
+@testset "jacobsen" begin
+    # real input
+    fs = 100
+    t = range(0, 5, step = 1/fs)
+    fr = 28.3
+    sr = cos.(2π*fr*t .+ π/4.2)
+    f_est_real = jacobsen(sr, fs)
+    @test isapprox(f_est_real, fr, atol = 0.5)
```

Comment:
In the complex case we have a tighter bound:
```suggestion
    @test isapprox(f_est_real, fr, atol = 1e-2)
```

But if I'm not mistaken, the frequency resolution of the underlying FFT is 0.2 here and hence simple peak-picking should be within 0.1, so `atol` should definitely be lower than that, but I'm not sure what one can expect.

Reply:
This was a side effect of the off-by-one error in `jacobsen`. I've re-written these tests and added some more. For `jacobsen`, now `atol = 1e-5` is used.